templates/commands/render_action_append.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"strings"
+
+	"github.com/abcxyz/abc/templates/model"
+)
+
+func actionAppend(ctx context.Context, ap *model.Append, sp *stepParams) error {
+	path, err := parseAndExecuteGoTmpl(ap.Path.Pos, ap.Path.Val, sp.inputs)
+	if err != nil {
+		return err
+	}
+
+	with, err := parseAndExecuteGoTmpl(ap.With.Pos, ap.With.Val, sp.inputs)
+	if err != nil {
+		return err
+	}
+
+	if !ap.SkipEnsureNewline.Val {
+		// TODO: would we rather just cast to bytes than take dependency?
```

Comment:
@drevell Thoughts? Seems like a style question. Technically casting to byte array would be faster, but harder to read.